dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdExtensionConfiguration.kt

```diff
@@ -47,12 +47,30 @@ open class DdExtensionConfiguration(
     var mappingFilePath: String? = null
 
     /**
-     * DO NOT SET THIS UNLESS YOU KNOW WHAT IT IS.
+     * Short aliases to use for package prefixes. Allows to replace, for example,
+     * androidx.appcompat with something shorter, reducing the size of the mapping file. Key is
+     * the prefix to replace, value is the replacement. Note, that these short aliases will be also
+     * in the deobfuscated stacktrace instead of the original prefixes. Sample usage:
+     *
+     * ```
+     *  "androidx.appcompat" to "axapp",
+     *  "androidx.work" to "axw",
+     *  "java.lang" to "jl",
+     *  "kotlin.collections" to "kc"
+     *  ...
+     * ```
+     *
+     * This property is not inherited, meaning in the particular variant configuration it should
+     * be declared explicitly, even if it exists in the root configuration.
```

Comment:
Is there a limitation preventing us to inherit this for variant? IMHO this would make it tedious to copy the same conf in all variants.

Reply:
The motivation behind that is the following:

* Not necessarily a particular variant needs that, it may have `mapping.txt` which simply fits the limit, so we shouldn't blindly apply what was declared in the root configuration (in the end introducing short aliases impacts the readability of the retraced stacks).
* If we say that variant should inherit this property, then what is the approach for inheritance: do we replace the mapping? Or do we merge what is declared in the variant with what is declared in the root configuration?

So considering that I prefer to not have an inheritance. In the end if user needs to share some mapping between variants, they can simply introduce a method which adds it with the format and merging strategy they want.